docs/deploy/security_group_management.md

```diff
@@ -0,0 +1,31 @@
+In the context of the AWS LB controller, we can classify security groups into two categories:
+
+1. Frontend SG
+2. Backend SG
+
+## Frontend SG
+
+Frontend security groups control the clients that can access the load balancer. The frontend security groups can be configured via the `alb.ingress.kubernetes.io/security-groups` annotation on the ingress resource. If the annotation is not specified, controller will auto-create one security group per load balancer to allow the traffic from `inbound-cidrs` to `listen-ports`.
+
+## Backend SG
+
+Backend security group controls the traffic between the load balancer and the EC2 instances or the ENIs. This SG is attached to the load balancer and is used as traffic source in the ENI/Instance SG rules. The backend security group is shared between multiple load balancers.
+
+The controller flag `--enable-backend-security-group` (default `true`) is used to enable/disable the shared backend SG. The flag `--backend-security-group` (default empty) is used to pass in the SG to use as a shared backend SG. If it is empty, the controller will auto-generate a SG with the following name and tags -
```

Comment:
Please spell out "security group". It's not that long.

Reply:
Perhaps document what happens if the enable flag is turned off?

Reply:
The shared backend security group should probably be a separate paragraph, if not a separate subsection.